src/caselawclient/models/documents/__init__.py

```diff
@@ -347,18 +347,13 @@ def enrich(self) -> bool:
         """
         Request enrichment of a document, if it's sensible to do so.
         """
-        if self.can_enrich:
+        if self.enriched_recently is False:
+            print("Enrichment requested")
             self.force_enrich()
             return True
+        print("Enrichment not requested as document was enriched recently")
         return False
 
-    @cached_property
-    def can_enrich(self) -> bool:
```

Comment:
A breaking change to the API, but appears safe.

Reply:
ah yeah added line in changelog, but did check it is unused in eui, pui